test/stripe_test.rb

```diff
@@ -33,4 +33,20 @@ class StripeTest < Test::Unit::TestCase
       Stripe.max_network_retries = old
     end
   end
+
+  should "pass the stripe-account in the header" do
```

Comment:
Could you also rename this one to "makes requests with the Stripe-Account header"?